testrunner/execute_test.go

```diff
@@ -134,31 +82,35 @@ func TestRunTests_brokenImport(t *testing.T) {
 			t.Errorf("Broken import test run - unexpected suffix in line: %s, want: %s", lines[i], expectedSuffix)
 		}
 	}
+}
 
-	output := &testReport{
-		Status:  statErr,
-		Version: 2,
-		Message: res,
+func TestRunTests_RuntimeError(t *testing.T) {
+	input_dir := "./testdata/practice/runtime_error"
+	cmdres, ok := runTests(input_dir)
+	if !ok {
+		fmt.Printf("runtime error test expected to return ok: %s", cmdres.String())
 	}
-	btr, err := json.MarshalIndent(output, "", "\t")
+
+	output := getStructure(cmdres, input_dir, 2)
```

Comment:
This is minor and unrelated to the PR so feel free to ignore. I wonder if we should make the version optional and default it to 2 in `getStructure()`, to remove the magic number here?

Reply:
I was always told to only use optional parameters if absolutely necessary as they are quite hacky in Go. I fixed the issue you mentioned by creating a version constant and passing that instead of the version number.

Reply:
> I was always told to only use optional parameters if absolutely necessary as they are quite hacky in Go. I fixed the issue you mentioned by creating a version constant and passing that instead of the version number.

Yes, I think you are right, that's the better solution here. Thanks for fixing, looks good!